R/data-storage-log-file.R

```diff
@@ -32,162 +26,98 @@ DataStorageLogFile <- R6::R6Class( # nolint object_name_linter
     #' @description
     #' Initialize the data storage class
     #' @param log_file_path string with path to JSON log file user actions
-    #' @param session_file_path string with path to JSON log file for the session details
 
-    initialize = function(
-      log_file_path, session_file_path
-    ) {
+    initialize = function(log_file_path) {
       super$initialize()
-      logger::log_debug("path to file: {log_file_path}", namespace = "shiny.telemetry")
-      private$connect(log_file_path = log_file_path, session_file_path = session_file_path)
-    },
-
-    #' @description Insert new data
-    #' @param values list of values to write to database
-    #' @param insert_time boolean flag that indicates if `time` parameters
-    #' should be added automatically
-    #' @param bucket path to log file; defaults to `log_file_path` used when initialized
-
-    insert = function(
```

Comment:
`insert` and `read_data` was removed in favor of implementing this in parent class.